R/produce_gpx.R

```diff
@@ -0,0 +1,44 @@
+#' Save collar locations to disk as a GPX file
+#'
+#' @param id_col Quouted name of the ID column that signals the individual identifier.
+#' @param lat_col Quouted name of the latitude column.
+#' @param lon_colQuouted name of the longitude column.
+#' @param dt_col Quouted name of the date/time column with class POSIXct.
+#' @param x A dataframe containing collar data with column "dt" that can be coerced into class "Date"
+#' @param dir The directory to save the GPX file output. Default is here::here().
+#' @param file_name The name of the GPX file created with this function.
+#' @param crs The coordiante refernce system associated with the collar data in the supplied data frame.
+#' @param ... Additional arguments to pass to sf::st_write()
+#'
+#' @return The original data frame passed to the function.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#'   produce_gpx(collar_data, start = Sys.date() - 30)
+#' }
+produce_gpx <- function(x,
+                        id_col = "id",
```

Comment:
I think it would be good to try and keep to everything quoted or unquoted. We started with unquoted arguments, but I am open to discussion on this point.